supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc

```diff
@@ -75,6 +75,17 @@ Use the hyphenated "fault-tolerant" when using it as an adjective.
 
 *See also*:
 
+//Add FedRAMP terminology Issue #395
+[[fedora-project]]
+==== image:images/yes.png[yes] FedRAMP® (noun)
+*Description*: The Federal Risk and Authorization Management Program (FedRAMP®) is managed by the FedRAMP Program Management Office. Use the ® symbol at first usage in a document. For additional information, see the link:https://www.fedramp.gov/assets/resources/documents/FedRAMP_Branding_Guidance.pdf[FedRAMP Brand Guide].
```

Comment:
Question: Should it be spelled out upon first use, or not? (Either way, I'd recommend mentioning that here)

A few suggestions too

```suggestion
*Description*: The _Federal Risk and Authorization Management Program (FedRAMP®)_ is managed by the FedRAMP Program Management Office. Use the registered trademark symbol (®) at first usage in a document. For more information, see the link:https://www.fedramp.gov/assets/resources/documents/FedRAMP_Branding_Guidance.pdf[FedRAMP Brand Guide].
```